common/src/main/java/com/mx/path/core/common/exception/PathRequestException.java

```diff
@@ -70,9 +70,15 @@
  * <p>Example:
  *
  * <pre>{@code
- *     throw new AccessorUserException("Identity unknown", "Username and/or password did not match system", PathResponseStatus.UNAUTHORIZED)
- *       .withReason("Bad password")
- *       .withCode("1401");
+ *   public class FailedLoginException extends AccessorUserException {
+ *     public FailedLoginException(String message, String userMessage) {
+ *       super(message)
+ *     }
+ *   }
+ *
+ *   throw new FailedLoginException("Identity unknown", "Username and/or password did not match system", PathResponseStatus.UNAUTHORIZED)
```

Comment:
Minor: The usage of `FailedLoginException` provides a status, but the constructor doesn't take one.

Reply:
Fixed.